en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-1100-to-320.md

```diff
@@ -307,7 +307,12 @@ versioning in the migrated 3.2.0 setup. Follow the instructions below to disable
 ### Step 1 - Migrate the API Manager configurations
 
 !!! warning
-    Do not copy the entire configuration files from the current version of WSO2 API Manager to the new one, as the configuration model has been changed and now all the configurations are done via a single file (`deployment.toml`). Instead, redo the configuration changes in the new configuration file. For more information, see the [Configuration Catalog]({{base_path}}/reference/config-catalog).
+    Do not copy the entire configuration files from the current version of WSO2 API Manager to the new one, as the configuration model has been changed and now all the configurations are done via a single file (`deployment.toml`). Instead, redo the configuration changes in the new configuration file.
+
+!!! note
+    
+    - For more information on the configurations in the new configuration model, see the [Configuration Catalog]({{base_path}}/reference/ConfigCatalog).
```

Comment:
Shouldn't the file/directory name 'ConfCatalog' change to have all simple lettering?